aepp/configs.py

```diff
@@ -166,17 +175,18 @@ def configure(
         sandbox : OPTIONAL : If not provided, default to prod
         connectInstance : OPTIONAL : If you want to return an instance of the ConnectObject class
         environment : OPTIONAL : If not provided, default to prod
-        auth_code : OPTIONAL : If an authorization code is used directly instead of generating via JWT
+        scopes : OPTIONAL : The scope define in your project for your API connection. Oauth V2, for clients and customers.
+        auth_code : OPTIONAL : If an authorization code is used directly instead of generating via JWT. Oauth V1 only, for adobe internal services.
     """
     if not org_id:
         raise ValueError("`org_id` must be specified in the configuration.")
     if not client_id:
         raise ValueError("`client_id` must be specified in the configuration.")
     if not secret:
         raise ValueError("`secret` must be specified in the configuration.")
-    if (auth_code is not None and (path_to_key is not None or private_key is not None)) \
-            or (auth_code is None and path_to_key is None and private_key is None):
-        raise ValueError("either `auth_code` needs to be specified or one of `private_key` or `path_to_key`")
+    if (scopes is not None and (path_to_key is not None or private_key is not None)) \
```

Comment:
I'm not sure that is entirely correct - for internal services we can authenticate using just client ID, client secret, and permanent auth code, there is no notion of scopes or it may be optional.